src/lib_modules.mli

```diff
@@ -33,6 +33,8 @@ val make
 
 val set_modules : t -> Module.Name_map.t -> t
 
+val version_installed : install_dir:Path.t -> t -> t
```

Comment:
The convention in dune is to always have `t` first.

Reply:
Yeah, that's what I mean. We usually avoid optional arguments, but those are added before the `t`.